src/inference/CMakeLists.txt

```diff
@@ -4,10 +4,6 @@
 
 set (TARGET_NAME "inference_engine")
 
-if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
-    # TODO: Add IMPLEMENT_OPENVINO_API to inference_engine_obj
```

Comment:
maybe we need to use `IMPLEMENT_OPENVINO_API ` instead of `IMPLEMENT_INFERENCE_ENGINE_API` here:
https://github.com/openvinotoolkit/openvino/blob/713b37cb251782333040f9798ada0989bc9b8823/src/inference/include/openvino/runtime/common.hpp#L18-L29
?